manifests/pin.pp

```diff
@@ -4,7 +4,7 @@
 define apt::pin(
   $ensure          = present,
   $explanation     = undef,
-  $order           = undef,
+  $order           = 50,
```

Comment:
I'm not sure how.  Nobody can use the module without setting a pin order when $order = undef. If you try then you get this error message:

```
Error: Could not retrieve catalog from remote server: Error 400 on SERVER: apt::setting priority must be an integer or a zero-padded integer on node vagrant.box
Warning: Not using cache on failed catalog
Error: Could not retrieve catalog; skipping run
```

Which is confusing since the problem comes from "order" and not "priority" and from apt::pin and not apt::setting.  I had to read the source to figure out what was going on.

So I could update the docs to show that order is a required value for apt::pin, but it's not reflected in any examples:

```
apt::pin { 'karmic': priority => 700 }
apt::pin { 'karmic-updates': priority => 700 }
apt::pin { 'karmic-security': priority => 700 }
```

It's mentioned in the apt::pin documentation but only to say it's an optional value and the default is 50.

Sorry to slip this in.  I honestly thought it was a minor change that was probably what everyone intended.  If this is truly not the case then I could update the documentation as stated, or leave it to someone who knows better what apt::pin is supposed to be doing. :)

I'll upload some tests in a couple of minutes.